### PR TITLE
Ukrainian: translate 'Version' and 'Last update'

### PR DESCRIPTION
**Pull request by hedrok**

### CODE REVIEW

File: lang-uk.conf [hunk 1]
```diff
@@ -26,9 +26,8 @@ appendix-caption=Додаток
 manname-title=НАЗВА
 
 [footer-text]
-#TODO: Translation of 'Version' and 'Last updated'.
-Version {revnumber}{basebackend-xhtml11?<br />}{basebackend-xhtml11=<br>}
-Last updated
+Версія {revnumber}{basebackend-xhtml11?<br />}{basebackend-xhtml11=<br>}
+Востаннє оновлено
```
Comment (elextr) [inline lang-uk.conf:30]: This should have `{docdate} {doctime}` after it.

**Review by elextr — CHANGES_REQUESTED** [review]

**Comment by hedrok** [comment]
Are you sure?.. I've checked lang-en, lang-de, lang-fr, none of them has this after various translations of "Last update".
Only lang-pl.conf has "{docdate} {doctime}", all others (including lang-uk) use "template::[footer-date]" which is defined in html4.conf and html5.conf as "{docdate} {doctime}" with some conditions.
Should I change lang-pl?.. I think "template::[footer-date]" is better.
What should I do? :)

**Comment by elextr** [comment]
@hedrok you are right, the change to remove the attributes was added earlier this year

But the lang-pl was added for the first time two days before the change that removed the attributes, so it was not in that change set. Oh well.  Has been fixed separately.

**Review by elextr — APPROVED** [review]

**Comment by hedrok** [comment]
Thank you for merging, but see my comment to your changes of lang-pl.conf please

**Pull request merged**